tf_quant_finance/black_scholes/approximations/__init__.py

```diff
@@ -0,0 +1,25 @@
+# Lint as: python3
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Approximations to the black scholes formula."""
+
+from tf_quant_finance.black_scholes.approximations.american_option import adesi_whaley
+from tensorflow.python.util.all_util import remove_undocumented  # pylint: disable=g-direct-tensorflow-import
+
+_allowed_symbols = [
+    'adesi_whaley',
+    'american_option',
```

Comment:
If you want expose american_option you need
from tf_quant_finance.black_scholes.approximations import american_option

and have a separate init file for american_option
This is probably a better thing to do

Reply:
Make a new directory for american_option? Super happy to make that change at some point - maybe worth waiting until we have more things in the approximation bucket so it isn't unnecessarily nested? will be a good practice to see how well i've understood these build configs now :D